.evergreen/run-mongosh-integration-tests.sh

```diff
@@ -12,9 +26,13 @@ npm pack | tee npm-pack.log
 npm cache clear --force || true
 npm i -g npm@8.x || true
 
-export TARBALL_FILENAME="$(tail -n1 npm-pack.log)"
-cd /tmp
-git clone --depth=10 https://github.com/mongodb-js/mongosh.git
-cd mongosh
-export REPLACE_PACKAGE="mongodb:${PROJECT_DIRECTORY}/${TARBALL_FILENAME}"
+TARBALL_FILENAME="$(tail -n1 npm-pack.log)"
+
+MONGOSH_DIRECTORY="/tmp/$TASK_ID"
+git clone --depth=10 -b misc-changes-for-node-driver-ci https://github.com/baileympearson/mongosh.git $MONGOSH_DIRECTORY
```

Comment:
Just a mental note that this line will need to change once the mongosh PR merges.

Reply:
fixed! mongosh PR merged.